go/oauth2l/main.go

```diff
@@ -47,7 +59,7 @@ func main() {
 		return
 	}
 
-	commands := map[string]func(*oauth2client.Token){
+	commands := map[string]func(*sgauth.Token){
 		"fetch":  fetch,
 		"header": header,
 		"token":  token,
```

Comment:
The python version also has test and info commands. Please add them.

Reply:
I have a follow-up PR to address this.

This PR I would like to focus on using the new Google Authenticator and keep the tool as-is. In next PR I will update the tool to match the python version.